### PR TITLE
[PREVIEW] ROC-4191 Add timeline and evidence to defendant's claim summary/details

### PR DESCRIPTION
**Pull request by andrewwa-kainos**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-4191

### Change description
The timeline entered by the claimant will be shown to the defendant in the claim summary of the first-contact journey, and in the claim details view when logged in and responding.

Iff evidence was entered by the claimant, this is will also be shown on the same two views.

### Work checklist

- [ ] Unit tests added where applicable
- [x] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [x] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/features/first-contact/views/claim-summary.njk [hunk 1]
```diff
@@ -19,6 +21,18 @@
       <p class="bold form-group-related">{{ t('Reason for claim:') }}</p>
       <p>{{ claim.claimData.reason }}</p>
 
+      <div class="form-group">
+        <p><span class="bold-small">{{ t('Timeline') }}</span></p>
+        {{ timelineSummary(claim.claimData.timeline.getPopulatedRowsOnly()) }}
+      </div>
+
+      {% if evidenceRows and 0 < evidenceRows | length %}
```
Comment (timja) [inline src/main/features/first-contact/views/claim-summary.njk]: could you do, `evidenceRow.length > 0`?

File: src/test/http-mocks/claim-store.ts [hunk 2]
```diff
@@ -90,6 +91,13 @@ export const sampleClaimObj = {
   }
 }
 
+export const sampleClaimDataWithEvidence = {
+  claim: {
+    ...sampleClaimObj.claim,
+    evidence: { rows: [{ type: EvidenceType.PHOTO.value, description: 'my photo evidence' }] }
```
Comment (timja) [inline src/test/http-mocks/claim-store.ts]: Is it worth doing this? i would just inline this into the route, that's what we would normally do for adding a field

File: src/main/features/first-contact/views/claim-summary.njk (hunk 1 above)
Comment (andrewwa-kainos) [reply src/main/features/first-contact/views/claim-summary.njk]: Yes

**Comment by andrewwa-kainos** [comment]
Merge is pending PO review

File: src/main/views/macro/summaryTable.njk [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{% macro evidenceSummary(evidence) %}
+  <table class="mobile-table">
+    <thead>
+      <tr>
+        <th class="bold-small tcol-half-width">{{ t('Type') }}</th>
```
Comment (andrewwa-kainos) [inline src/main/views/macro/summaryTable.njk]: new `tcol-half-width` class hasn't been committed yet

Comment (andrewwa-kainos) [reply src/main/views/macro/summaryTable.njk]: Fixed with commit 17.

**Comment by andrewwa-kainos** [comment]
Shown to Marc James with approval

**Pull request merged**